core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -933,6 +953,17 @@ class KafkaZkClient(zooKeeperClient: ZooKeeperClient, isSecure: Boolean) extends
     createResponse.resultException.foreach(e => throw e)
   }
 
+  private def logDirFailureEventZkData(brokerId: Int): String = {
+    val LogDirFailureEvent = 1
+    Json.encode(Map("version" -> LogDirEventNotificationHandler.Version, "broker" -> brokerId, "event" -> LogDirFailureEvent))
```

Comment:
This is already available in ZkData.LogDirEventNotificationSequenceZNode.encode method